iac-extensions/docker/src/test/resources/checks/MisusedUnsetCheck.dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM scratch
+
+ENV SECRET_VALUE=password SECRET_USER=admin
+
+# Noncompliant@+1 {{Use the ARG instruction or set & unset environment variable in a single layer.}}
+RUN ./script -p $SECRET_VALUE && unset SECRET_VALUE
+#                                      ^^^^^^^^^^^^
+# Noncompliant@+1
+RUN ./script -p $SECRET_VALUE && unset -v SECRET_VALUE
+#                                         ^^^^^^^^^^^^
+# Noncompliant@+2
+# Noncompliant@+1
```

Comment:
Well, there are two variables being unset and an issue is raised for each in the current implementation. However, this can be marked as `# Noncompliant@+1 2`